README.md

```diff
@@ -38,10 +35,18 @@ The project is released under the terms of the [MPL 2.0](https://mozilla.org/MPL
 
 ## Contact
 
+[GitHub Discussions](https://github.com/sogno-platform/dpsim/discussions) - Ask questions, share ideas, and get community support.
```

Comment:
I dont think its good two have two points of contact here.

Which one should new users approach first?
I would say we should choose only one of them.